.github/workflows/test.yml

```diff
@@ -69,11 +73,25 @@ jobs:
         with:
           node-version: 'lts/*'
           cache: npm
+      - name: Extract Aerie backend docker tag from PR body
+        # look in the PR body for eg. the string ___REQUIRES_AERIE_PR___=9999, extract the number & save to env var
+        # if backend PR is labeled correctly, it will publish a docker tag called 'pr-9999' to use in tests
+        if: ${{ contains(env.PR_BODY, '___REQUIRES_AERIE_PR___=') }}
+        run: |
+          echo "AERIE_IMAGE_TAG=pr-$(echo $PR_BODY | sed -n 's/.*___REQUIRES_AERIE_PR___=\"\([0-9]\+\)\".*/\1/p')" >> $GITHUB_ENV
+      - name: Extract Aerie gateway docker tag from PR body
+        # look in the PR body for eg. the string ___REQUIRES_AERIE_PR___=9999, extract the number & save to env var
+        # if backend PR is labeled correctly, it will publish a docker tag called 'pr-9999' to use in tests
+        if: ${{ contains(env.PR_BODY, '___REQUIRES_GATEWAY_PR___=') }}
+        run: |
+          echo "GATEWAY_IMAGE_TAG=pr-$(echo $PR_BODY | sed -n 's/.*___REQUIRES_GATEWAY_PR___=\"\([0-9]\+\)\".*/\1/p')" >> $GITHUB_ENV
       - name: Start Services (Aerie)
         run: |
-          docker compose -f docker-compose-test.yml up -d
+          echo "AERIE_IMAGE_TAG: $AERIE_IMAGE_TAG"
+          echo "GATEWAY_IMAGE_TAG: $GATEWAY_IMAGE_TAG"
+          docker compose -f docker-compose-test.yml up -d --quiet-pull
           docker images
-          docker ps -a
+          docker ps -a --no-trunc
```

Comment:
Why was the `--no-trunc` flag added?

Reply:
Just a small thing i noticed while looking at workflow logs... `docker ps` truncates information to fit in the table (namely the command used to run the container), thought it would be more useful to have the full version in logs